atlasdb-api/src/main/java/com/palantir/atlasdb/config/LeaderConfig.java

```diff
@@ -55,9 +52,6 @@ public File acceptorLogDir() {
     @JsonProperty("lockCreator")
     @Value.Default
     public String lockCreator() {
-        log.warn("Lock creator not specified! "
```

Comment:
What is our official recommendation about specifying a lockCreator? If we recommend setting one, then I think a warning here is appropriate. 

(Personally I prefer explicit over implicit)